apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -306,6 +336,11 @@ userView { setState, state: { logger } } user = React.fragment
           , description: "Byt lösenord"
           , className: passwordChangeClass
           }
+      , formatIconLink
+          { href: "/fakturor"
```

Comment:
I still think it would be better to open this in the same the tab, as it's going to load a second instance of mitt konto. Now that we're using react router, even the browser history works as expected.